benchmarks/bench.sh

```diff
@@ -34,7 +34,7 @@ COMMAND=
 BENCHMARK=all
 DATAFUSION_DIR=${DATAFUSION_DIR:-$SCRIPT_DIR/..}
 DATA_DIR=${DATA_DIR:-$SCRIPT_DIR/data}
-CARGO_COMMAND=${CARGO_COMMAND:-"cargo run --release"}
+CARGO_COMMAND=${CARGO_COMMAND:-"cargo run --release --locked"}
```

Comment:
either remove --locked or add a code comment why it should be here

Reply:
[684c7df](https://github.com/apache/datafusion/pull/14483/commits/684c7df6ecf9c7579654b68cb733a943173961e2)